spec/site_prism/page_objects/pages/cases/new/osar_page_subject_details.rb

```diff
@@ -40,7 +40,7 @@ def fill_in_case_details(params={})
             set_date_of_birth kase.date_of_birth
 
             choose('offender_sar_subject_type_offender', visible: false)
-            choose('offender_sar_flag_for_disclosure_specialists_no', visible: false)
+            choose('offender_sar_flag_as_high_profile_false', visible: false)
           end
 
           def set_date_of_birth(date_of_birth)
```

Comment:
This file should be named `offender_sar_page_subject_details.rb`

Reply:
Javid has a ticket for renaming all the remaining "osar"s - suggest we do it in one go rather than piecemeal